src/main/java/frc/robot/utils/AbsoluteEncoderChecker.java

```diff
@@ -0,0 +1,48 @@
+package frc.robot.utils;
+
+import edu.wpi.first.math.filter.MedianFilter;
+
+/** Checks Absolute Encoders for Noise */
+public class AbsoluteEncoderChecker {
+    private double[] lastFive;
+    private int index;
+    private MedianFilter medianFilter;
+
+    public AbsoluteEncoderChecker() {
+        this.lastFive = new double[5];
+        this.index = 0;
+        this.medianFilter = new MedianFilter(5);
+    }
+
+    public void addReading(double lastReading) {
+        lastFive[index] = lastReading;
+        index ++;
+        if (index == 5) {
+            index = 0;
+        }
+    }
+
+    /** @return true if all of the last five absolute encoder readings were equal, false if otherwise*/
+    public boolean checkLastFive() {
+        for (int i = 1; i < lastFive.length; i ++) {
+            if (lastFive[i] != lastFive[0]) {
+                return false;
+            }
+        }
+        return true;
+    }
```

Comment:
Maybe change to "encoder connected" and return `true` if any of them are different (flipped from current logic)? Then the sendable text can be like "Deploy encoder connected"

Reply:
Like change the comment to "return true if encoder connected" or change the name of the function to encoderConnected?